components/exercises/chapter-5-oscar/Flattening/Flattening.tsx

```diff
@@ -0,0 +1,67 @@
+import { getFlattenedArray } from './getFlattenedArray';
+
+const arrays = [
```

Comment:
Try to avoid names of variables that are super general like `arrays` or `objects` etc, when the file grows you might not know what `arrays` refers to. 

What about `iconsArrays` or `arraysOfIcons`?

Reply:
refactored to arraysOfIcons thank you